decidim-initiatives/app/views/decidim/initiatives/initiatives/_committee_members.html.erb

```diff
@@ -1,73 +1,50 @@
-<fieldset>
-  <legend><%= t ".title" %></legend>
+<div class="block">
+  <span class="font-semibold text-xl"><%= t ".link" %></span>
+  <span class="help-text mb-2">
+    <%= t ".invite_to_committee_help" %>
+  </span>
 
-  <div class="card-section">
-    <div class="row column">
-      <%= t ".invite_to_committee_help" %>
-    </div>
-
-    <div class="row column">
-      <div class="card card--list">
-        <div class="card--list__item">
-          <div class="card--list__text">
-            <div>
-              <h6 class="card--list__heading heading6 lowercase" id="committee_link">
-                <%= decidim_initiatives.new_initiative_committee_request_url(current_initiative) %>
-              </h6>
-            </div>
-          </div>
-          <div class="card--list__data">
-            <%= icon_link_to "clipboard", "#", t(".invite_to_committee_help"), class: "card--list__data__icon invite-users-link" %>
-          </div>
-        </div>
-      </div>
+  <div class="initiative__form__committee">
+    <span>
+      <%= decidim_initiatives.new_initiative_committee_request_url(current_initiative) %>
+    </span>
+    <div class="ml-auto fill-secondary inline-block">
+      <button type="button" class="button button__sm button__text-secondary" data-clipboard-copy="#urlShareLink">
+        <%= render_committee_tooltip %>
+      </button>
+      <input id="urlShareLink" type="text" class="!hidden" value="<%= decidim_initiatives.new_initiative_committee_request_url(current_initiative) %>" readonly>
```

Comment:
is `!hidden` a valid class name?

Reply:
Yes, it is to add an `!important` to the class. If you use an invalid class, the webpacker throws an error